statsmodels/tools/decorators.py

```diff
@@ -4,7 +4,26 @@
 from statsmodels.compat.python import get_function_name
 import warnings
 
-__all__ = ['resettable_cache', 'cache_readonly', 'cache_writable']
+__all__ = ['resettable_cache', 'cache_readonly', 'cache_writable',
+           'deprecated_alias']
+
+
+def deprecated_alias(old_name, new_name, remove_version=None):
+    msg = '%s is a deprecated alias for %s' % (old_name, new_name)
```

Comment:
Should modern string formatting be used `'{0}'.format(var)`?

Reply:
I think the pytest precedent is that its OK for me to take my time getting comfortable with that.

Reply:
I would autopep all of SM, PRs be damned.

Reply:
> I would autopep all of SM, PRs be damned.

My foolish hope is that jpkt will delegate design-decision-free fixups to you and I so we can a) clear out the PR backlog and b) clean up the flaketastrophe.